politeiawww/user/localdb/localdb.go

```diff
@@ -465,6 +465,16 @@ func (l *localdb) RegisterPlugin(p user.Plugin) error {
 	return nil
 }
 
+// EmailHistoriesGet is a stub to satisfy the user MailerDB interface.
+func (l *localdb) EmailHistoriesGet(_ []string) (map[string]user.EmailHistory, error) {
```

Comment:
Doing so breaks some user tests that expect nil errors being returned, but instead get the "not implemented" error. This implies in some ugly hacks to get the tests back working which I wanted to avoid. I'm thinking we can return nil errors here but explain it better on the comments that `localdb` is only used for testing and that the user layer is getting rewritten shortly so this implementation was not needed. 

If this is really not desirable, I think the best would be for me to just implement them on localdb also. What do you think?

Reply:
opted to implement them in localdb since its not a lot of extra work 👍